sdk/computerecommender/Azure.ResourceManager.ComputeRecommender/api/Azure.ResourceManager.ComputeRecommender.net8.0.cs

```diff
@@ -0,0 +1,102 @@
+        string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.ComputeRecommender.Models.PlacementScore>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.ComputeRecommender.Models.PlacementScore>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+    }
+    public partial class ResourceSize : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.ComputeRecommender.Models.ResourceSize>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.ComputeRecommender.Models.ResourceSize>
+    {
+        public ResourceSize() { }
+        public string Sku { get { throw null; } set { } }
+        protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer, System.ClientModel.Primitives.ModelReaderWriterOptions options) { }
+        Azure.ResourceManager.ComputeRecommender.Models.ResourceSize System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.ComputeRecommender.Models.ResourceSize>.Create(ref System.Text.Json.Utf8JsonReader reader, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        void System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.ComputeRecommender.Models.ResourceSize>.Write(System.Text.Json.Utf8JsonWriter writer, System.ClientModel.Primitives.ModelReaderWriterOptions options) { }
+        Azure.ResourceManager.ComputeRecommender.Models.ResourceSize System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.ComputeRecommender.Models.ResourceSize>.Create(System.BinaryData data, System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.ComputeRecommender.Models.ResourceSize>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+        System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.ComputeRecommender.Models.ResourceSize>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
+    }
+    public partial class SpotPlacementScoresContent : System.ClientModel.Primitives.IJsonModel<Azure.ResourceManager.ComputeRecommender.Models.SpotPlacementScoresContent>, System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.ComputeRecommender.Models.SpotPlacementScoresContent>
+    {
+        public SpotPlacementScoresContent() { }
+        public bool? AvailabilityZones { get { throw null; } set { } }
+        public int? DesiredCount { get { throw null; } set { } }
+        public System.Collections.Generic.IList<string> DesiredLocations { get { throw null; } }
```

Comment:
are the items in this list `AzureLocation`s?
If so, please use `@@alternateType` to change the type of this property to `azureLocation[]` in your client.tsp

Reply:
Hi @ArcturusZhang, I have created a PR here: https://github.com/Azure/azure-rest-api-specs/pull/37226 Does this look correct?